zerver/actions/user_settings.py

```diff
@@ -146,6 +148,7 @@ def do_change_user_delivery_email(
         modified_user=user_profile,
         event_type=AuditLogEventType.USER_EMAIL_CHANGED,
         event_time=event_time,
+        extra_data={"old_value": original_email, "new_value": new_email},
```

Comment:
I think we have a `RealmAuditLog.OLD_VALUE` convention to use?

Reply:
Oh yeah, fixed - thanks for catching!